core/integration/features/auctions/opening-auction-price.feature

```diff
@@ -65,8 +65,8 @@ Feature: Set up a market, create indiciative price different to actual opening a
       | party7 | ETH/DEC19 | sell | 1      | 11000 | 0                | TYPE_LIMIT | TIF_GFA | t7-s-1    |
     And the parties should have the following margin levels:
       | party  | market id | maintenance | search | initial | release |
-      | party1 | ETH/DEC19 | 11200       | 12320  | 13440   | 29120   |
-      | party2 | ETH/DEC19 | 10900       | 11990  | 13080   | 28340   |
+      | party1 | ETH/DEC19 | 11200       | 12320  | 13440   | 15680   |
```

Comment:
I see maintenance margin staying the same but release is changing - why? (probably an easy answer, but I am slow)

Reply:
If you look at the last file in the diff (`core/risk/margins_calculation.go`) you'll see that there used to be a hack in place to prevent a margin release during auction. An auction state object has been added since, so I'm now just doing properly using that object.